pkg/manifest/anaconda_installer.go

```diff
@@ -455,6 +462,8 @@ func (p *AnacondaInstaller) dracutStageOptions() *osbuild.DracutStageOptions {
 			"dmsquash-live",
 			"convertfs",
 		}...)
+	default:
+		panic("Unknown AnacondaInstallerType in dracutStageOptions")
```

Comment:
Thanks for adding this, maybe we could include the unkonwn type in the panic, i.e. `panic(fmt.Errorf("unknown AnacondaInstallerType %v in dracutStageOptions", p.Type))`  ? (and by making it an error we prepare for a potential future with less panicing :)

Reply:
`p.Type` is an int, right?  Not very useful, but we could add it and mention it's an "invalid enum value".